src/app/feedback/feedback.component.html

```diff
@@ -0,0 +1,31 @@
+<h1 i18n>Add feedback</h1>
+<form [formGroup]="feedbackForm" (ngSubmit)="addfeedback(feedbackForm.value)">
+  <div>
+    <label i18n>Is your feedback Urgent?</label>
+    <mat-radio-group  formControlName ="isUrgent">
+      <mat-radio-button value="true" i18n>Yes</mat-radio-button>
+      <mat-radio-button value="false" i18n>no</mat-radio-button>
+    </mat-radio-group>
+  </div>
+  <div>
+    <label i18n>Feedback Type:</label>
+    <mat-radio-group formControlName="feedbackType">
+      <mat-radio-button value ="Question" i18n>Question</mat-radio-button>
+      <mat-radio-button value ="Bug" i18n>Bug</mat-radio-button>
+      <mat-radio-button value ="Suggestion" i18n>Suggestion</mat-radio-button>
+    </mat-radio-group>
+  </div>
+  <div>
+    <label i18n>Comment:</label>
+    <mat-form-field>
+      <textarea matInput placeholder="Your Feedback *" formControlName="feedbackMsg"></textarea>
```

Comment:
no required attribute added. Also material form can accommodate '*" on its own. So no need for adding extra symbol on label